terraform/aws/components/pss/data_secrets.tf

```diff
@@ -33,3 +33,7 @@ data "aws_secretsmanager_secret" "postgres_gpc_facade_password" {
 data "aws_secretsmanager_secret_version" "postgres_psdbowner_username" {
   secret_id = data.aws_secretsmanager_secret.postgres_psdbowner_username.id
 }
+
+data "aws_secretsmanager_secret" "gpc-consumer_sds_apikey" {
+  name = var.secret_name_sds_apikey
```

Comment:
What was the thinking behind making this a variable? Looks like the rest of this file uses hardcoded values.

Reply:
Oh, I see we use this variable already within the GP2GP component, makes sense.